package.json

```diff
@@ -46,6 +46,7 @@
     "recharts": "2.15.4",
     "sonner": "^2.0.7",
     "tailwind-merge": "^3.3.1",
+    "vaul": "^1.1.2",
```

Comment:
What is vaul? Is this something that is required and gets installed as part of shadcn drawer?